assets/model_monitoring/components/src/genai_token_statistics_compute_metrics/metrics_processor.py

```diff
@@ -167,10 +167,16 @@ def calculate_metrics(self, attributes: dict, span_type: str, root_input: str, r
                 "output": root_output
             })
 
+    def get_value_from_attributes(self, attributes, keys):
```

Comment:
add a default_value instead of a hard code 0 in case no attribute found, can default the default_value to 0 though.

Reply:
seems this is not addressed?